### PR TITLE
RPi4 Integration

### PR DESCRIPTION
**Pull request by nihemstr**
- Added layer support for RPi4
- Added machine configurations for build
- Added some more documentation on necessary build items
- Made build script a bit more efficient

### CODE REVIEW

File: yocto/config-templates/raspberrypi4-64/local.conf.sample [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+#
+# Enable support for automatically running a local hash equivalence server and
+# instruct bitbake to use a hash equivalence aware signature generator. Hash
+# equivalence improves reuse of sstate by detecting when a given sstate
+# artifact can be reused as equivalent, even if the current task hash doesn't
+# match the one that generated the artifact.
+#
+# A shared hash equivalent server can be set with "<HOSTNAME>:<PORT>" format
+#
+#BB_HASHSERVE = "auto"
```
Comment (jw-msft) [inline yocto/config-templates/raspberrypi4-64/local.conf.sample:262]: I found uncommenting these 2 can speed things up

File: scripts/build.sh [hunk 2]
```diff
@@ -189,6 +189,8 @@ export ADUC_PRIVATE_KEY_PASSWORD=$ADUC_KEY_DIR/priv.pass
 # Remove the conf dir before sourcing oe-init-build-env
 if [[ $CLEAN == "true" ]]; then
     rm -rf $BUILD_DIR/conf
+    # cp $ROOT_DIR/yocto/config-templates/raspberrypi4-64/local.conf $BUILD_DIR/conf
```
Comment (jw-msft) [inline scripts/build.sh:192]: these are commented-out. remove or planning to uncomment them in the future?

**Review by jw-msft — APPROVED** [review]
Approved with comments

File: scripts/build.sh (hunk 2 above)
Comment (nihemstr) [reply scripts/build.sh:192]: Cleanup PR is in the works :)

**Pull request merged**